Makefile

```diff
@@ -42,9 +46,11 @@ run-flask: ## Run flask
 run-celery: ## Run celery
 	. environment.sh && celery \
 		-A run_celery.notify_celery worker \
-		--pidfile="/tmp/celery.pid" \
-		--loglevel=INFO \
-		--concurrency=4
+		--uid=$(shell id -u easuser) \
+		--pidfile=/tmp/celery.pid \
+		--loglevel=WARNING \
```

Comment:
Any reason we've moved from INFO to WARNING?

Reply:
The CloudWatch logs for the scheduled celery tasks is very noisy (admittedly most of the output has WARN level anyway) - we moved to a less verbose log level to try to reduce the volume of output written to the logs.